nautobot/circuits/templates/circuits/circuit_create_or_update.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'generic/object_edit.html' %}
```

Comment:
Are these going to effectively be just a `git mv circuit_edit.html circuit_create_or_update.html` (etc.) or are there actual differences in the files? Might make the diffs more self-evident if you `git rm` the templates that aren't being used any more here.